Models/TodoFilterVm.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Efcore_demo.Models;
+
+public class TodoFilterVm
+{
+    public string TodoSearch { get; set; }
```

Comment:
Use explicit filters rather then single filter param. use [date,title,status]..

Reply:
i will surly add this too